meteor-app/imports/api/members/rejects.js

```diff
@@ -0,0 +1,10 @@
+// rejects.js
+import { Mongo } from 'meteor/mongo'
+
+//
+// A place to put the member records we couldn't import
+//
+
+const Rejects = new Mongo.Collection('rejects')
```

Comment:
This collection is to hold any records that get rejected during the import process.